apps/frontend/src/pages/profile/profile.tsx

```diff
@@ -0,0 +1,35 @@
+import { Loader, PageLayout } from "~/libs/components/components.js";
+import { DataStatus } from "~/libs/enums/data-status.enum.js";
+import { useAppSelector } from "~/libs/hooks/hooks.js";
+
+import { EditUserForm } from "./components/components.js";
+import styles from "./styles.module.css";
+
+const Profile = (): JSX.Element => {
+	const { authenticatedUser, dataStatus } = useAppSelector(({ auth }) => auth);
+
+	if (!authenticatedUser) {
+		return <></>;
+	}
+
+	const isLoading =
+		dataStatus === DataStatus.PENDING || dataStatus === DataStatus.IDLE;
+
+	return (
+		<PageLayout>
+			<div className={styles["profile-content"]}>
+				<h1 className={styles["label"]}>Profile</h1>
```

Comment:
should be title, it will be renamed in other places in another PR
```suggestion
				<h1 className={styles["title"]}>Profile</h1>
```